mteb/models/sentence_transformers_models.py

```diff
@@ -391,7 +391,11 @@
     use_instructions=False,
     public_training_code=None,
     public_training_data=None,
-    training_datasets=None,
+    training_datasets={
+        "MS MARCO": ["train"],
+        "Google Natural Questions": ["train"],
+        "Community QA": ["train"],
+    },
```

Comment:
Can you move datasets to variable and update like? 
```suggestion
    training_datasets={
            "MSMARCO": ["train"],
    "MSMARCOHardNegatives": ["train"],
    "NanoMSMARCORetrieval": ["train"],
    "MSMARCO-PL": ["train"],  # translation not trained on
    "mMARCO-NL": ["train"],  # translation not trained on
    "NQ": ["train"],
    "NQ-NL": ["train"],  # translation not trained on
    "NQHardNegatives": ["train"],
    "NanoNQRetrieval": ["train"],
    "NQ-PL": ["train"],  # translation not trained on

        "Community QA": ["train"],
    },
```

Reply:
If the translations are not trained on, why are we adding them?

Reply:
We've discussed it in pr's somewhere. These datasets are correlated, so it is likely that the model results will be biased